test/k8sT/Hubble.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/annotation"
+	. "github.com/cilium/cilium/test/ginkgo-ext"
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("K8sHubbleTest", func() {
```

Comment:
why is this assigned into a var?

Reply:
This is what the Ginkgo [docs](http://onsi.github.io/ginkgo/) recommend:
> The var _ = ... trick allows us to evaluate the Describe at the top level without having to wrap it in a func init() {}